rules/framework.bzl

```diff
@@ -25,7 +25,7 @@ def apple_framework(name, apple_library = apple_library, **kwargs):
 
 def _find_framework_dir(outputs):
     for output in outputs:
-        prefix = output.path.split(".framework/")[0]
+        prefix = output.path.rsplit(".framework/", 1)[0]
```

Comment:
can give an example of how this changes what's inside the `prefix` variable?

Reply:
It only would if `output.path` contained multiple instances of `.framework`, in which case now it will return the innermost framework, instead of the outermost

Reply:
so `a.framework/b/c.framework/c` to give `a.framework/b/c.framework`